File: Python/Week_1/day5_logical_operators.py
# Day 5 - Logical Operators in Python

age = 20
has_id = True

print("age >= 18 and has_id:", age >= 18 and has_id) 
print("age >= 18 or has_id:", age >= 18 or has_id)      
print("not has_id:", not has_id)                        

# Real-world example
age2 = 16
has_permission = True

if age2 >= 18 or has_permission:
    print("Allowed to enter")
else:
    print("Not allowed")

File: Java/Week_1/Day5LogicalOperators.java
public class Day5LogicalOperators {
    public static void main(String[] args) {
        
        int age = 20;
        boolean hasId = true;
        
        System.out.println("age >= 18 && hasId: " + (age >= 18 && hasId));   
        System.out.println("age >= 18 || hasId: " + (age >= 18 || hasId));   
        System.out.println("!hasId: " + (!hasId));                             
        
        // Real-world example
        int age2 = 16;
        boolean hasPermission = true;
        
        if (age2 >= 18 || hasPermission) {
            System.out.println("Allowed to enter");
        } else {
            System.out.println("Not allowed");
        }
    }
}